index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile first</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <div class="header1">
+
+        <h3 class="logo">My Logo</h3>
+        <img src="./img/hamburger.png" alt="hamburger_image">
+
+    </div>
```

Comment:
- [ ] [REQUIRED] Good job so far 👍  Please change the `div` tag to `header` tag because `header` tag is more semantic and the logo should be has a link (`a` tag) 😃